examples/persistent-volume-provisioning/README.md

```diff
@@ -243,20 +243,25 @@ parameters:
     group: "root"
     quobyteConfig: "BASE"
     quobyteTenant: "DEFAULT"
+    createQuota: "False"
+    tenantNamespaceMapping: "True"
+    createTenants: "True"
 ```
 
 [Download example](quobyte/quobyte-storage-class.yaml?raw=true)
 <!-- END MUNGE: EXAMPLE quobyte/quobyte-storage-class.yaml -->
 
-* **quobyteAPIServer** API Server of Quobyte in the format http(s)://api-server:7860
+* **quobyteAPIServer** API Server of Quobyte in the format `http(s)://api-server:7860`.
 * **registry** Quobyte registry to use to mount the volume. You can specify the registry as <host>:<port> pair or if you want to specify multiple registries you just have to put a comma between them e.q. <host1>:<port>,<host2>:<port>,<host3>:<port>. The host can be an IP address or if you have a working DNS you can also provide the DNS names.
 * **adminSecretName** secret that holds information about the Quobyte user and the password to authenticate against the API server. The provided secret must have type "kubernetes.io/quobyte".
 * **adminSecretNamespace** The namespace for **adminSecretName**. Default is `default`.
 * **user** maps all access to this user. Default is `root`.
 * **group** maps all access to this group. Default is `nfsnobody`.
-* **quobyteConfig** use the specified configuration to create the volume. You can create a new configuration or modify an existing one with the Web console or the quobyte CLI. Default is `BASE`
-* **quobyteTenant** use the specified tenant ID to create/delete the volume. This Quobyte tenant has to be already present in Quobyte. For Quobyte < 1.4 use an empty string `""` as `DEFAULT` tenant. Default is `DEFAULT`
-* **createQuota** if set all volumes created by this storage class will get a Quota for the specified size. The quota is set for the logical disk size (which can differ from the physical size e.q. if replication is used). Default is ``False
+* **quobyteConfig** use the specified configuration to create the volume. You can create a new configuration or modify an existing one with the Web console or the quobyte CLI. Default is `BASE`.
+* **quobyteTenant** use the specified tenant name to create/delete the volume. This Quobyte tenant has to be already present in Quobyte. For Quobyte < 1.4 use an empty string `""` as `DEFAULT` tenant. Default is `DEFAULT`.
+* **createQuota** if set all volumes created by this storage class will get a Quota for the specified size. The quota is set for the logical disk size (which can differ from the physical size e.q. if replication is used). Default is `False`.
+* **tenantNamespaceMapping** if set the plugin will map all volume requests to the Quobyte tenant with the same name like the namespace of the PVC. For the best experience use this option with **createtenants**. This option is mutually exclusive to **quobyteTenant** if both options are set this option will be used. Default is `false`.
```

Comment:
if set to true, the plugin will use the namespace of the PVC as the Quobyte Tenant instead of **quobyteTenant**. For the best experience, this option is used with with **createTenants**. Default is 'false'.